stl/inc/random

```diff
@@ -1116,27 +1118,43 @@ protected:
     _Post_satisfies_(this->_Idx == 0)
 
         void _Refill_lower() { // compute values for the lower half of the history array
-        size_t _Ix;
-        for (_Ix = 0; _Ix < _Nx - _Mx; ++_Ix) { // fill in lower region
-            _Ty _Tmp       = (this->_Ax[_Ix + _Nx] & _HMSK) | (this->_Ax[_Ix + _Nx + 1] & _LMSK);
-            this->_Ax[_Ix] = (_Tmp >> 1) ^ (_Tmp & 1 ? _Px : 0) ^ this->_Ax[_Ix + _Nx + _Mx];
-        }
+        if constexpr (_M_mod_n > 0) {

```

Comment:
I am wondering, whether it would be clearer to flip and check for `_M_mod_n == 0`